taccsite_cms/signal_processor.py

```diff
@@ -0,0 +1,29 @@
+from haystack.signals import BaseSignalProcessor
+from django.db import models
+from django.core.management import call_command
+from cms import signals
+
+
+class RealtimeSignalProcessor(BaseSignalProcessor):
+    """
+    A signal processor to make a call to the Django Haystack
+    `rebuild_index` management command when a CMS document is
+    published, unpublished, or deleted.
+
+    Usage:
+        This signal processor hooks into Haystack via the
+        `HAYSTACK_SIGNAL_PROCESSOR` Django setting.
+    """
```

Comment:
Would you please use the word "search" somewhere? Maybe

```suggestion
    """
    Calls the Django Haystack `rebuild_index` management
    command (to rebuild search index) when a CMS document
    is published, unpublished, or deleted.

    Usage:
        This signal processor hooks into Haystack via the
        `HAYSTACK_SIGNAL_PROCESSOR` Django setting.
    """
```

_It's a bit technical, which at least will keep people from breaking it. Perhaps they could bite down on `rebuild_index` and follow the trail. But just letting them now that this is search-related is the hope._